examples/app-fixtures.sh

```diff
@@ -17,5 +17,5 @@ flask users create -a info@inveniosoftware.org --password 123456
 # Upload sample records
 echo '{"title": "Public", "body": "test 1", "public": 1}' > public.json
 echo '{"title": "Private", "body": "test 2", "public": 0}' > private.json
-flask index put demo example -b public.json
-flask index put demo example -b private.json
+flask index put 'records-default-v1.0.0' example -b public.json
```

Comment:
This being a shell script is highly recommended (if you put then, the worst case is that they were not needed, if you don't it might become a big issue), though for this case is not really needed.